.babelrc

```diff
@@ -0,0 +1,10 @@
+{
+  "presets": [
+    [
+      "@babel/preset-env",
+      {
+        "useBuiltIns": "entry"
```

Comment:
I'm not really sure what we should use here in babelrc, I think I just took an example from their site